lib/src/flutter_google_places.dart

```diff
@@ -3,10 +3,16 @@ library flutter_google_places.src;
 import 'dart:async';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
+import 'package:geocoder/geocoder.dart';
 import 'package:google_maps_webservice/places.dart';
 import 'package:http/http.dart';
 import 'package:rxdart/rxdart.dart';
 
+
+
+
+var globalEnableZipCode = false;
```

Comment:
it does not have to be global+public, consider making it private by renaming it to `_globalEnableZipCode` or making it non-global